scripts/services_test.sh

```diff
@@ -36,7 +36,8 @@ run_test () {
     download_artifacts "$REPO_NAME" "${TEST_NAME}-GCC_ARM-${TARGET}-${SHA}" "$BARE_TOKEN" ${TEST_NAME}.hex
     set -x
     cp ${TEST_NAME}.hex "$MOUNTPOINT"
-    sleep 20
+    sync
```

Comment:
We've been evaluating other solutions to flashing: mbed-flasher, mbed-flsh and openocd. We will revisit this once we got more definitive results.